### PR TITLE
NTBS-2414 Add other disease site text to legacy extract

### PR DESCRIPTION
**Pull request by benjimarshall**
### Description
Add the `SiteDescription` text stored for 'other' disease sites in NTBS to the legacy extract.

### Testing
Tested locally using ETS test data. This change has not been deployed anywhere yet.

### CODE REVIEW

**Review by aptemus — APPROVED** [review]
Looks good

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql [hunk 1]
```diff
@@ -64,6 +64,17 @@ BEGIN TRY
...
+	--Add in the OtherExtraPulmonarySite text
+	UPDATE le
+		SET OtherExtraPulmonarySite = Descriptions.SiteDescription
+	FROM [dbo].[Record_LegacyExtract] le
+		INNER JOIN (
+			SELECT NotificationId,
+					SiteDescription
+				FROM [$(NTBS)].[dbo].[NotificationSite]
+				WHERE SiteId = 17
+		) AS Descriptions ON Descriptions.NotificationId = le.NotificationId
```
Comment (JoshCadney) [inline source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql:76]: This is probably just a personal preference, but I'd find this clearer without the nested select, i.e.
```
UPDATE le
SET OtherExtraPulmonarySite = ns.SiteDescription
FROM [dbo].[Record_LegacyExtract] le
INNER JOIN [$(NTBS)].[dbo].[NotificationSite] ns ON ns.NotificationId = le.NotificationId
WHERE ns.SiteId = 17
```

Comment (JoshCadney) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql:76]: That said, I don't know which would be more performant

**Review by JoshCadney — APPROVED** [review]
Looks good :-)

File: source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql (hunk 1 above)
Comment (benjimarshall) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql:76]: Ah that definitely would be tidier and shouldn't impact performance at all. I have just merged this though, so I'll make a quick follow on PR

Comment (benjimarshall) [reply source/dbo/Stored Procedures/Power BI Reporting/uspGenerateReportingLegacySitesOfDisease.sql:76]: Follow-on PR at #69 @JoshCadney

**Pull request merged**